README.md

```diff
@@ -220,6 +225,73 @@ When working with large-scale input data, it is important to create large enough
 
 TODO: Add an example for running on Google Cloud.
 
+## Frequently asked questions (FAQs)
+A small selection of FAQs:
+
+**Question: Is there something similar to `tf.keras.Sequential`?**
```

Comment:
We don't want the README to be too long. Could you make the questions link to the FAQ instead of inlining the answer?

Reply:
Okay will link to the questions in the actual FAQ

Reply:
Actually @jheek there seems to be no easy way to directly link it to the questions in the actual FAQ, so can you suggest what questions I can remove to make the README shorter?

Reply:
Sorry for the late reply, I lost track of this issue.

I think it is better tot keep the FAQ questions out of the README if they cannot be inlined.

Reply:
Updated: Removed questions, retain link to the original FAQ